packages/twenty-server/src/workspace/utils/compute-object-target-table.util.ts

```diff
@@ -0,0 +1,9 @@
+import { ObjectMetadataInterface } from 'src/metadata/field-metadata/interfaces/object-metadata.interface';
+
+export const computeObjectTargetTable = (
```

Comment:
@charlesBochet An utils already exist called `createCustomColumnName`, but this one only prefix the string by taking a string in parameter.
I think we should get rid of one of them, to only keep one in the code base 🙂